charts/bpa/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: bpa
 description: The Business Partner Agent allows to manage and exchange master data between organizations.
 type: application
 
-version: 0.7.1
+version: 0.7.3
```

Comment:
same as in the other PR , use a pre-release for 0.8 ( 0.8-something)

Reply:
or is it a fix here?

Reply:
if we anyway are nearby the next sprint release , fix is not required because the PO's can use the next sprint release

Reply:
I have set it to 0.8-alphax